ajax/libs/jshashtable.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jshashtable.js",
+  "version": "3.0",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Tim Down",
+    "email": "tim@timdown.co.uk"
+  },
+  "homepage": "http://timdown.co.uk/jshashtable/",
+  "filename": "hashtable.min.js",
+  "description": "A standalone JavaScript implementation of hash table with arbitrary keys",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/timdown/jshashtable.git"
+  },
+  "keywords": [
+    "javascript",
+    "hash",
+    "hastable",
+    "hashset"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/timdown/jshashtable.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "+(hashset| hashtable)*.+(js|css|map)"
```

Comment:
Please remove the space between `|` and `h`.
We don't have to add extra space.
Also this pattern can't get the `hashtable*` files.
Many thanks